README.md

```diff
@@ -15,6 +15,12 @@ This module requires the following modules/libraries:
 
 ## Installation
 
+Existing Solr fields configuration from Drupal 7 can be migrated by dropping the
```

Comment:
This documentation is probably not needed. If someone is following the standard Drupal upgrade paths it should just work and if they are off the beaten path then the docs are out of scope.